source/MaterialXTest/Document.cpp

```diff
@@ -32,6 +32,13 @@ TEST_CASE("Document", "[document]")
     REQUIRE(constant->getNamePath() == "nodegraph1/node1");
     REQUIRE(constant->getNamePath(nodeGraph) == "node1");
 
+    // Test getting elements by path
+    REQUIRE(doc->getElementByPath("") == doc);
+    REQUIRE(doc->getElementByPath("nodegraph1") == nodeGraph);
+    REQUIRE(doc->getElementByPath("nodegraph1/node1") == constant);
+    REQUIRE(nodeGraph->getElementByPath("") == nodeGraph);
+    REQUIRE(nodeGraph->getElementByPath("node1") == constant);
```

Comment:
Should probably add a test with a non-empty path which is not found.

Reply:
Sure, I can add that.

Reply:
And for the comment below (since I can't reply to it directly), I can add this to both repos.